File: problem-solving/BOJ/1543.py
text = input()
findText = input()
result = 0
i = 0
while i < len(text):
    index = i
    for j in range(0, len(findText)):
        if index >= len(text):
            i += 1
            break
        if text[index] == findText[j]:
            if j == len(findText) - 1:
                result += 1
                i += len(findText)
                break
            index += 1
            continue
        i += 1
        break
print(result)

File: problem-solving/BOJ/1543.cpp
#include "bits/stdc++.h"
using namespace std;
string a, b;
int main(void){
    ios::sync_with_stdio(0);
    cin.tie(0);
    getline(cin, a); //공백 포함 문자열은 getline으로 해결
    getline(cin, b);
    int result = 0;
    for (int i = 0; i < a.length();) {
        int index = i;
        for (int j = 0; j < b.length(); j++) {
            if(a[index] == b[j]) {
                if(j == b.length() - 1) {
                    result++;
                    i += b.length();
                    break;
                }
                index++;
                continue;
            }
            i++;
            break;
        }
    }
    cout << result;
}